src/git_changelog/build.py

```diff
@@ -441,10 +441,10 @@ def _bump(self, version: str) -> None:
                     target=last_version.planned_tag,
                 )
 
-    def _fix_single_version(self) -> None:
+    def _fix_single_version(self, version: str | None) -> None:
         last_version = self.versions_list[0]
         if len(self.versions_list) == 1 and last_version.planned_tag is None:
-            planned_tag = "0.1.0"
+            planned_tag = version if version and version not in {"auto", "major", "minor", "patch"} else "0.1.0"
```

Comment:
What do you think of this:

```suggestion
            planned_tag = "0.1.0" if version in {"auto", "major", "minor", "patch", "", None} else version
```

Just to make it shorter :sweat_smile:

Reply:
yes, agree. Shorter is better.

Reply:
I reverted the change. `make check` does not like it:

```
✓ (python3.10)  Checking code quality
✗ (python3.10)  Type-checking (1)
  > mypy --config-file config/mypy.ini src tests duties.py scripts
  src/git_changelog/build.py:448: error: Incompatible types in assignment (expression has type "str | None", variable has type "str")  [assignment]
  src/git_changelog/build.py:449: error: Unsupported operand types for + ("str" and "None")  [operator]
  src/git_changelog/build.py:449: note: Right operand is of type "str | None"
  src/git_changelog/build.py:450: error: Argument 2 to "replace" of "str" has incompatible type "str | None"; expected "str"  [arg-type]
```